package.json

```diff
@@ -20,10 +20,7 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^3.5.2",
-    "intl": "~1.2.5",
-    "jed": "^1.1.1",
-    "react-intl": "^2.8.0"
+    "@theforeman/vendor": "3.8.0-intl.0"
```

Comment:
I deployed this version to `npm` so it will be easier to test.
It should still be tested against https://github.com/theforeman/foreman-js/pull/81 locally to make sure both work well.